docs/conf/berlin/2025/news/one-month-out.md

```diff
@@ -8,19 +8,21 @@ og:image: _static/conf/images/headers/{{shortcode}}-{{year}}-opengraph.jpg
    :tags: {{shortcode}}-{{year}}
 ```
 
-# Only 1 Month Until Write the Docs {{name}}
+# Only one Month Until Write the Docs {{name}}
 
 Write the Docs {{name}} is officially just over 1 month away on October 27-28! Whether you're a programmer, tech writer, designer, project manager, or developer advocate, we have talks and a community for you at Write the Docs {{name}}.
 
 ## Buy Your Tickets
 
 Still need a ticket? Now is a great time to purchase your ticket at one of our varying price levels.
 
-- Student or Unemployed Tickets: €75
-- Independent Tickets: €275
-- Corporate Tickets: €575
+- In-person Student or Unemployed Tickets: €75
+- In-person Independent Tickets: €275
+- In-person Corporate Tickets: €575
 - Virtual Tickets: €30-140
```

Comment:
This weirds me out that we don't split virtual tix in the same way as in person.